### PR TITLE
Added Fairy Tale - Artificial General Intelligence Solutions

### PR DESCRIPTION
**Pull request by freeman42x**
Also linked to GitHub Sponsors and OpenCollective

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ A curated list of awesome Haskell people profiles you can support by sponsoring
 * [@tarleb — Albert Krewinkel](https://github.com/sponsors/tarleb)
 * [@vrom911 — Veronika Romashkina](https://github.com/sponsors/vrom911)
 
+## [OpenCollective](https://opencollective.com/)
+
+* [Fairy Tale - Artificial General Intelligence Solutions](https://github.com/fairy-tale-agi-solutions)
```
Comment (chshersh) [inline README.md]: Thanks for the addition! But it's not clear from the link how to sponsor your work. Could you provide a direct link to your open collective profile instead?

Comment (freeman42x) [reply README.md]: Sure

Comment (freeman42x) [reply README.md]: Done

**Review by chshersh — APPROVED** [review]
Thanks!

**Review by vrom911 — APPROVED** [review]
Awesome! Thank you :+1:

**Pull request merged**